WooCommerce/Classes/ViewRelated/Products/ProductsListViewModel.swift

```diff
@@ -23,6 +23,14 @@ class ProductListViewModel {
         selectedProducts.count
     }
 
+    var variableProductsCount: Int {
```

Comment:
nit: Should we name this `selectedVariableProductsCount` to make it clear that it's specifically counted selected variable products and not all variable products in the list?

Reply:
Agree! Renamed in 837bd24.